packages/jupyter-ai/jupyter_ai/chat_handlers/learn.py

```diff
@@ -118,13 +118,16 @@ async def process_message(self, message: HumanChatMessage):
         if args.verbose:
             self.reply(f"Loading and splitting files for {load_path}", message)
 
-        await self.learn_dir(
-            load_path, args.chunk_size, args.chunk_overlap, args.all_files
-        )
-        self.save()
-
-        response = f"""🎉 I have learned documents at **{load_path}** and I am ready to answer questions about them.
-        You can ask questions about these docs by prefixing your message with **/ask**."""
+        try:
+            await self.learn_dir(
+                load_path, args.chunk_size, args.chunk_overlap, args.all_files
+            )
+        except Exception as e:
+            response = f"""Learn documents in **{load_path}** failed. Additional packages needed: {str(e)}."""
```

Comment:
In case we want to support other unknown exceptions, we should add multiple except blocks. The current response is targeting import errors, and might be confusing to users for other exceptions.
```suggestion
        except ModuleNotFoundError as e:
            response = f"""Learn documents in **{load_path}** failed. Additional packages needed: {str(e)}."""
        except Exception as e:
            response = f"""Learn documents in **{load_path}** failed. {str(e)}."""
```